Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -300,8 +301,32 @@ its :php:`__invoke()` method will be called:
 
 Read :ref:`how to configure dependency injection in extensions <dependency-injection-in-extensions>`.
 
+Existing event listeners can be overridden by custom implementations. For example,
+:file:`EXT:news/Configuration/Services.yaml` could listen on the event
+:php:`TYPO3\CMS\Frontend\Event\ModifyHrefLangTagsEvent`:
 
+..  literalinclude:: _ServicesOverrideBase.yaml
+    :language: yaml
+    :caption: EXT:my_extension/Configuration/Services.yaml
+
+If you want to replace this event listener with your custom implementation, your extension can
+achieve this by specifying:
+
+..  literalinclude:: _ServicesOverrideCustom.yaml
+    :language: yaml
+    :caption: EXT:my_extension/Configuration/Services.yaml
 
+Make sure that you set the :yaml:`identifier` attribute to exactly the string that the
+original implementation uses. If that attribute is not mentioned specifically in the original
```

Comment:
```suggestion
original implementation uses. If the identifier is not mentioned specifically in the original
```

Event listeners can be tagged by a PHP attribute in v13. Better use not that word for not confusing the reader ...

Reply:
Ack, done